seqtyping/seq_typing.py

```diff
@@ -456,7 +456,13 @@ def reads_subcommand(args):
 
     rematch_script = include_rematch_dependencies_path()
 
-    utils.required_programs({'rematch.py': ['--version', '>=', '4.0.1']})
+    utils.required_programs({
+        'rematch.py': ['--version', '>=', '4.0.1'],
+        'bowtie2-build': ['--version', '>=', '2.2.9'],
+        'bowtie2': ['--version', '>=', '2.2.9'],
+        'samtools': ['--version', '==', '1.3.1'],
```

Comment:
Is there some issue with more recent versions of `samtools` and `bcftools` which forces us to use a strict version (`==`) here? They are in version 1.21 already, version 1.3.1 is from 2016-04-22, quite dated now.